boomerang-dfp-timing.js

```diff
@@ -188,7 +211,10 @@ BOOMR.plugins.DFPTiming = {
 		}
 
 		if (!impl.initialized) {
-			// Add googletag event listeners to record timing
+			// Read any cached events that fired before boomerang loaded
+			impl.readCachedEvents();
+
+			// Add googletag event listeners to record timing for any subsequent events
 			impl.attachEvents();
```

Comment:
Good point.  I refactored to just read from the cached event log on page_unload instead of re-applying the event listeners.